examples/emr-eks-app/test/emr-eks-app.test.ts

```diff
@@ -0,0 +1,17 @@
+// import * as cdk from 'aws-cdk-lib';
+// import { Template } from 'aws-cdk-lib/assertions';
+// import * as EmrEksApp from '../lib/emr-eks-app-stack';
+
+// example test. To run these tests, uncomment this file along with the
+// example resource in lib/emr-eks-app-stack.ts
+test('SQS Queue Created', () => {
+//   const app = new cdk.App();
```

Comment:
can you either uncomment and provide testing or remove this file if not providing tests?